CODEOWNERS

```diff
@@ -192,24 +192,21 @@ lib/posix/                               @pfalcon
 kernel/device.c                          @ramakrishnapallala @nashif
 kernel/idle.c                            @ramakrishnapallala @nashif
 samples/bluetooth/                       @sjanc @jhedberg @Vudentz
-samples/boards/quark_se_c1000/power*/    @ramakrishnapallala @nashif
 samples/display/                         @vanwinkeljan
 samples/gui/                             @vanwinkeljan
 samples/net/                             @jukkar @tbursztyka @pfalcon
 samples/net/dns_resolve/                 @jukkar @tbursztyka @pfalcon
-samples/net/http_server/                 @jukkar @tbursztyka
 samples/net/lwm2m_client/                @mike-scott
-samples/net/mbedtls_sslclient/           @jukkar
 samples/net/mqtt_publisher/              @jukkar @tbursztyka
-samples/net/coap_client/                 @rveerama1
-samples/net/coap_server/                 @rveerama1
+samples/net/sockets/coap_client/         @rveerama1
```

Comment:
I guess, @mike-scott should be added to these, but I guess, he can do that in a separate PR.